client/angular/test/nightwatch/helpers/page_helper.coffee

```diff
@@ -73,5 +73,5 @@ module.exports = (test) ->
     test.acceptAlert()
     @pause()
 
-  waitFor: (selector, wait = 4000) ->
+  waitFor: (selector, wait = 20000) ->
```

Comment:
Eek, this is too long I think; a failing test suite will take forever if we keep this value; waitFor is run before any expectElement or expectText, so any failing test will take 20 seconds minimum, plus 2/3 retries....

Reply:
oh this bit was an accidental commit, thanks for seeing it. Testing locally needs a much longer time out.